### DISCUSSION TITLE
[Other] Fresh CapRover Install has issues

### DISCUSSION THREAD

**Discussion by tinjaw** [jonaswinkler/paperless-ng#1242] [Support]
More to follow. I just wanted to get this posted. I will add details when I have time.

I did a fresh install using CapRover. The first thing I am attempting to do is add a tag. (No documents yet.) I am getting a 403

`2021-08-16T12:43:47.324297013Z [2021-08-16 07:43:47,323] [WARNING] [django.request] Forbidden: /api/tags/`

**Comment by tinjaw** [comment]
I have time to work on this now. I just tried to create a tag named 'test'. I click save and nothing appears to happen. I check the server logs and see

'2021-08-19T01:29:34.750837320Z [2021-08-18 20:29:34,749] [WARNING] [django.request] Forbidden: /api/tags/'

I have never used paperless before. I saw it as a one-click CapRover app and decided to check it out. Any help getting it up and running would be appreciated.

**Comment by dangnhdev** [comment]
I fixed it by remove the PAPERLESS_COOKIE_PREFIX parameter. Not sure if this was an issue of Paperless-ng itself or Caprover.

**Reply by danielslyman** [reply]
This worked for me too.
